tests/integration/tc_150_1.c

```diff
@@ -0,0 +1,40 @@
+#include "../test.h"
+
+/**
+ * Tests the Keyboard input source through net_handler, SHM, and executor. 
+ */
+
+int main() {
+    start_test("keyboard_input", "keyboard_input", "", NO_REGEX);
+
+    connect_virtual_device("SimpleTestDevice", 20);
+
```

Comment:
I would suggest adding a `sleep(1)` after calling this function, as it can take a bit for the device to connect. From adding a couple print statements, I found that the device doesn't connect until after `TELEOP` started. (I posted a screenshot in the Slack thread, nearly 10 lines later)